test/WebSites/FiltersWebSite/Controllers/FormatFilterController.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Core.Filters;
+
+namespace FiltersWebSite
+{
+    [FormatFilter]
+    public class FormatFilterController : Controller
+    {   
+        public DummyClass GetProduct(int id)
+        {
+            return new DummyClass() { SampleInt = id };
+        }
+
+        [Produces("application/custom", "application/json", "text/json")]
+        public DummyClass ProducesMethod(int id)
+        {
+            return new DummyClass() { SampleInt = id }; ;
+        }
```

Comment:
would like to see a test case with overridden produces, just because we've spent so much time discussing it